health/service/src/main/java/com/cerner/beadledom/health/HealthDependency.java

```diff
@@ -35,10 +50,18 @@
   /**
    * Returns a Boolean to indicate if this check is a primary check for this service.
    */
+  @Deprecated
   public Boolean getPrimary() {
     return primary;
   }
 
+  /**
+   * Returns a Boolean to indicate if this dependency is a primary dependency for this service.
```

Comment:
Even simpler for boolean methods is something like `Returns true if this is a primary dependency; otherwise false.`

Reply:
Changed the boolean methods here: https://github.com/cerner/beadledom/pull/123/commits/95686b23b05f48a497bd6fac0a15acfb9712165b